test/core/test_project_status.py

```diff
@@ -281,20 +283,34 @@ def test_cache_fixes_update(self):
 
     def test_get_exceeded_thresholds(self):
         build = self.create_build('1')
-        testrun = build.test_runs.create(
-            environment=self.environment)
+        testrun = build.test_runs.create(environment=self.environment)
         testrun.metrics.create(name='metric1', suite=self.suite, result=3)
         testrun.metrics.create(name='metric2', suite=self.suite, result=2)
-        testrun.metrics.create(name='metric1', suite=self.suite, result=5)
+        testrun.metrics.create(name='metric3', suite=self.suite, result=5)
+
+        build_a = self.create_build('2')
+        testrun_a = build_a.test_runs.create(environment=self.environment_a)
+        testrun_a.metrics.create(name='metric4', suite=self.suite_a, result=3)
+        testrun_a.metrics.create(name='metric5', suite=self.suite_a, result=2)
+        testrun_a.metrics.create(name='metric6', suite=self.suite_a, result=7)
+
         status = ProjectStatus.create_or_update(build)
         MetricThreshold.objects.create(project=self.project,
-                                       name='metric1', value=4,
+                                       environment=self.environment,
+                                       name='/metric2', value=4,
                                        is_higher_better=True)
-
         thresholds = status.get_exceeded_thresholds()
+        self.assertEqual(len(thresholds), 0)
```

Comment:
This was supposed to be 1, no? Since metric threshold has `value=4, is_higher_better=True, name='/metric2'` and there's a testrun with a metric like `name='metric2', suite='/', result=2` 

Does it make sense?